packages/babel-check-binary-expressions/src/index.js

```diff
@@ -0,0 +1,8 @@
+export default function(left, right) {
+  // eslint-disable-next-line no-undef
+  if (left instanceof BigInt && right instanceof BigInt) {
```

Comment:
Would it make sense to use `window.BigInt` (or global.BigInt) to avoid the `no-undef` rule?